glibccheck-maven-plugin/src/test/java/io/netty/build/maven/GlibcVersionCheckMojoTest.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2020 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.build.maven;
+
+import org.apache.maven.plugin.MojoFailureException;
+import org.junit.Test;
+
+public class GlibcVersionCheckMojoTest {
+
+    private static final String OBJDUMP_OUTPUT = "\n" +
+            "/tmp/libnetty_transport_native_epoll_x86_64.so:     file format elf64-x86-64\n" +
```

Comment:
consider testing with output from macOS as well

Reply:
this is one for GLIBC which means you can only test files compiled on linux.